src/main/java/com/ucl/imaginethisserver/Conf/GsonConfig.java

```diff
@@ -19,6 +19,7 @@
     @Bean
     public Gson gson() {
         return new GsonBuilder()
+                .excludeFieldsWithoutExposeAnnotation()
```

Comment:
This project use Gson as the preferred JSON processor as configured in `application.properties` file `spring.mvc.converters.preferred-json-mapper=gson`

when adding this `.excludeFieldsWithoutExposeAnnotation()`, only fields annotated with `@Exposed` will be marshalled to JSON string. So if you add this line here, all other endpoints are broked since we do not annotate the fields of our `model` class with `@Exposed`.

However if we then manually add `@Exposed` annotation for all of our model classes, there will be too much boiler plate code. Thus, I recommend creating a custom gson annotation named `@Exclude` to exclude the fields that we don't want to be marshalled to JSON string, as mentioned [here](https://www.baeldung.com/gson-exclude-fields-serialization).
This should be a easy change

Reply:
Also, a lot of boilerplate getter/setter code can be simplified using `lombok` library, this library can make the class files easier to read. However, this is just my personal suggestion to use `lombok`

Reply:
The reason why I did this is because in file `FigmaComponents/Wireframe.java` I have an attribute:
```
@Expose(serialize = false)
private JsonArray children;
```
How would you do this, that I don't want the attribute to be included during serialization only deserialization? Accroding to [SO](https://stackoverflow.com/questions/43047823/gson-parameter-get-serialised-even-though-it-has-exposeserialize-false) this option is the only way...

Reply:
Yeah, I'll do what you said. It's even mentioned in the SO I mentioned above